src/Bridge/Doctrine/Common/Util/IdentifierManagerTrait.php

```diff
@@ -39,6 +41,7 @@ private function normalizeIdentifiers($id, ObjectManager $manager, string $resou
         $doctrineClassMetadata = $manager->getClassMetadata($resourceClass);
         $doctrineIdentifierFields = $doctrineClassMetadata->getIdentifier();
         $isOrm = interface_exists(EntityManagerInterface::class) && $manager instanceof EntityManagerInterface;
+        $isOdm = class_exists(DocumentManager::class) && $manager instanceof DocumentManager;
```

Comment:
Why the call to `class_exists`? If it doesn't exist, the subsequent check will return `false` anyways.